tests/scripts/generate_test_keys.py

```diff
@@ -99,9 +99,9 @@ def main() -> None:
     args = argparser.parse_args()
 
     output_file = args.output
-    # Remove output file if already existing.
+    # If the output file already exists, then we can quit (successfully)
```

Comment:
Yes, I agree. I designed it that way though because I was afraid that regenerating all the times could cause some extra unnecessary work for the CI. But I agree that is the safest approach.